src/Features/CSharp/Portable/Completion/CompletionProviders/NamedParameterCompletionProvider.cs

```diff
@@ -79,7 +79,8 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
                 return;
             }
 
-            if (token.IsMandatoryNamedParameterPosition())
+            var languageVersion = ((CSharpParseOptions)document.Project.ParseOptions).LanguageVersion;
+            if (languageVersion < LanguageVersion.CSharp7_2 && token.IsMandatoryNamedParameterPosition())
```

Comment:
Can you put a note that ideally we'd like to update this for C#7.2 and above so that if there is a named parameter present and it's *not* in its positional location, that named parameters would now be mandatory?